components/sidebar/sidebar_service.cc

```diff
@@ -35,6 +35,18 @@ namespace sidebar {
 
 namespace {
 
+// An array with the default builtin item types.
+constexpr SidebarItem::BuiltInItemType kDefaultBuiltInItemTypes[] = {
+    SidebarItem::BuiltInItemType::kBraveTalk,
+    SidebarItem::BuiltInItemType::kWallet,
+    SidebarItem::BuiltInItemType::kBookmarks,
+    SidebarItem::BuiltInItemType::kReadingList};
+
+// A list of preferred item types
+constexpr SidebarItem::BuiltInItemType kPreferredItemTypes[] = {
```

Comment:
Now that this is module-global, perhaps a better name is `kPreferredPanelOrder`

Reply:
Fixed.